node/setup.go

```diff
@@ -530,6 +530,7 @@ func startStateSync(
 //------------------------------------------------------------------------------
 
 var genesisDocKey = []byte("genesisDoc")
```

Comment:
```suggestion
//var genesisDocKey = []byte("genesisDoc")
```
Commenting it out to appease the linter, but we'll need it when implementing the upgrade path